pkg/module/go_runtime.go

```diff
@@ -0,0 +1,67 @@
+package module
```

Comment:
let's call this file according to the only function it's in: `prepare_env.go` instead of `go_runtime` which can be confusing as to what it means.

Reply:
i want to provide unified functions relative to go command execute in this file next.
by now, it also ok to name it as `prepare_env.go`, :)